src/test/suite/fixtures/recorded/languages/clojure/clearCall.yml

```diff
@@ -0,0 +1,23 @@
+languageId: clojure
```

Comment:
Hopefully these test cases are at least somewhat readable if you're unfamiliar with them 😅.  Basically they're all of the form "clear X", where "X" is a scope type such as "call", "list", "pair", etc.  The idea is that it deletes the matched scope, so you can see what it referred to.  

You can see the spoken form on line 4, and then check out the initial document contents on line 10, followed by the final document contents on line 16.  So you can see here that "call" (eg function call), matches a parenthesized expression that's unquoted